fix floating point error

Pietervanhalem opened the pull request.

uijl commented on openclsim/core.py:
```diff
@@ -1298,7 +1298,7 @@ def determine_route(self, origin, destination):
 
         for node in geom.keys():
-            if origin.x == geom[node].x and origin.y == geom[node].y:
+            if abs(origin.x - geom[node].x)<0.00000001 and abs(origin.y - geom[node].y)<0.00000001:
```
Could you solve it with [numpy.isclose](https://docs.scipy.org/doc/numpy/reference/generated/numpy.isclose.html)? That is a more elegant solution.
And remember the Black formatting.

uijl commented:
The black formatting test on CircleCI is broken, so please remove that form the yaml as well or continue on the branch that has this error included (https://github.com/TUDelft-CITG/OpenCLSim/tree/fix/movers)

Pietervanhalem commented:
I spoke to soon when I sad that I could make the node detection method more elegant. This only works if you have a list of nodes and not when the nodes are inside an object. I fixed the np.isclose issue and removed black. can you merge it? can you also make a new version on PIPY so that it works inside my docker container? thanks in advance!
